docs/README.md

```diff
@@ -1,6 +1,38 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the auth-service's schema:
+
+<p align="center">
+  <img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/spikerheado1234/auth-service/master/docs/ErDiagram.puml">
+</p>
+
+The following table depicts the types of each attribute in the `Consent` table, algonside their constraints:
+
+|Attribute| Represented Internal Type | Constraints|
+|:---------:|:---------------------------:|:------------:|
+|`id`|VARCHAR(36)|Unique - Primary Key|
+|`initiator_id`|VARCHAR(36)|Non-Null|
+|`participant_id`|VARCHAR(36)|Non-Null|
+|`credential_id`|TEXT|(Can be Null)|
+|`credential_type`|TEXT|(Can be Null)|
+|`credential_status`|TEXT|(Can be Null)|
+|`credential_payload`|TEXT|(Can be Null)|
+|`credential_challenge`|TEXT|(Can be Null)|
+
+The following table depicts the types of each attribute in the `Scope` table:
+
+|Attribute| Represented Internal Type | Constraints|
+|:---------:|:---------------------------:|:------------:|
+|`id`|VARCHAR(36)|Unique - Primary Key|
+|`consent_id`|VARCHAR(36)|Foreign Key referring to the `id` in the consent table|
+|`scope_account_type`|VARCHAR(36)|Non-Null|
```

Comment:
```suggestion
|`action`|`VARCHAR(36)`|Non-Null|
```

Reply:
This isn't really an account type, it's an action permitted on an account (e.g., `get balance` or `send money`)

Reply:
Modified data types and updated the PR.